flink-table/flink-table-common/src/main/java/org/apache/flink/table/factories/TableSinkFactory.java

```diff
@@ -98,6 +98,11 @@
 		 * {@code TableEnvironment} session configurations.
 		 */
 		ReadableConfig getConfiguration();
+
+		/**
+		 * @return true if input of the sink is bounded, otherwise it is a unbounded input.
```

Comment:
Could you take `Boundedness` as an example to improve the Javadoc a little? It's still not very clear what is bounded means here, and streaming may also produce bounded input.

Reply:
```
Now it depends on whether the {@link TableEnvironment} execution mode is batch.
In the future, the new sink interface will infer from input to source, see {@link Source#getBoundedness}.
```
What do you think?